inst/rmarkdown/templates/mdpi/resources/template.tex

```diff
@@ -125,21 +160,41 @@
 %\LSID{\url{http://}}
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-% Only for the journal Applied Sciences:
+% Only for the journal Applied Sciences
```

Comment:
Currently, I have added these as optional (documented) yaml fields. I'm hesitant to add more code to check journal title into the format function because (1) the maintenance overhead (I have no idea how often mdpi is changing their template); and (2) it is on the author to ensure they are including the required or optional components of a manuscript. It appears that these journal specific latex environments actually render no matter what journal is specified so it won't break the build process if they end up included.